closure/templates/closure_js_template_library.bzl

```diff
@@ -69,7 +79,13 @@ _closure_js_template_library = rule(
         ),
         "outputs": attr.output_list(),
         "globals": attr.label(allow_single_file = True),
-        "plugin_modules": attr.label_list(),
+        "plugin_modules": attr.string_list(
+            doc = "fully-qualified names of extension classes",
+        ),
+        "plugin_module_deps": attr.label_list(
+            doc = "java_library rules providing the specified plugin_modules",
+            providers = [JavaInfo],
+        ),
```

Comment:
IMO, it would be cleaner to create a `soy_plugin` rule that returns a provider with the `JavaInfo` and the class name, but, TBH, looking at `closure_Js_template_library`, it‘s probably not worth it (see #389 #424).

Reply:
Ah, I agree, that seems like a better setup. If it won't be superseded by your modernization work, I'll see about updating it to use that design.